### DISCUSSION TITLE
Mounting Solution

### DISCUSSION THREAD

**Discussion by LavenderFox2430** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#70] [Q&A]
I plan on purchasing one of these, however I can't exactly figure out how I'd go about installing this inside an exterior breaker box. There aren't any outlets around the breaker box and I'd rather not have to pay to have one installed. 

Does anyone have any ideas as to how someone can go about converting 120V AC to 9V AC within the breaker box itself?

**Comment by alexruffell** [comment]
I struggled with your same question so I will post what I am doing even though you are way past this point given when you posted the question.

The picture below is still work in progress...

![20220814_172726](https://user-images.githubusercontent.com/4006944/184558194-664c7d25-04a8-42be-91b1-459156654e2e.jpg)

I made a small board with 2 transformers. One powers the boards and the other just provides a reference for the other split-phase. In the picture I am taking measurements for the conduit that I need to add to run the CT clamp wires from the panel to the box. I opted to do everything outside the main panel due to lack of space, and it not being weatherproof. While it may be ok for breakers, I doubt it is any good for electronics in the long run. The box I picked is watertight so water should not be an issue but I will find out whether heat will be... given I live in Texas. The blueish wires hanging there are for the second board I am still waiting on. In the picture you can't see the large PCB wifi antenna stuck to the top of the box which will hopefully allow for a good connection.

**Comment by CircuitSetup** [comment]
Very nice! Thanks for sharing!

FYI, there is a chip temp parameter for the meter that you could potentially use to trigger an exhaust fan, if needed. Since the transformer won't be under much load, it shouldn't get too hot, though.
